bisdn/installer/install.sh

```diff
@@ -173,11 +206,11 @@ delete_bisdn_linux_msdos_partition()
     partprobe $blk_dev
 }
 
-# Creates a new partition for the BISDN Linux OS.
+# Creates a new msdos partition for the BISDN Linux OS.
```

Comment:
we should also update line 237 to echo that we create an msdos partition

Reply:
Done.